client/src/features/Catalogue/components/CatalogFilterDuration/CatalogFilterDuration.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import Grid from '@mui/material/Unstable_Grid2';
+import { FilterAccordion, OutlinedToggleButton } from '..';
+import { getProducts } from '../../../../store/slices/catalogueSlice/catalogueSlice';
+
+
+const CatalogFilterDuration = () => {
+  const dispatch = useDispatch();
+  const products = useSelector((state) => state.catalogue.products, shallowEqual);
+
+  const sorting = () => {
```

Comment:
the function name is not semantic function names have to be verbs reflecting the action the the function does.

Reply:
changes applied